api/SpeechSynthesisEvent.json

```
@@ -201,37 +201,53 @@
           "spec_url": "https://wicg.github.io/speech-api/#dom-speechsynthesisevent-elapsedtime",
           "support": {
             "chrome": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
```

So I think if something has not been updated it is correct to add a note and a bug link, if known.

---

PS NOTE, I am aware milliseconds is miss-spelled. I figure I'll fix that globally at the end. 

```suggestion
              "notes": "Units are milliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
```